packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/InputStatus/InputStatus.js

```diff
@@ -1,29 +1,36 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import styled from 'styled-components'
+import counterpart from 'counterpart'
+import en from './locales/en'
+
+counterpart.registerTranslations('en', en)
 
 export const StyledInputStatus = styled.span`
   font-size: 0.8em;
   opacity: 0.7;
   text-align: right;
 `
 
-// TODO Add Translations
 export default function InputStatus ({ count, tool }) {
   const minStyleColor = (count < tool.min) ? 'red' : ''
   const maxStyleColor = (count === tool.max) ? 'orange' : ''
+  const drawn = counterpart('InputStatus.drawn')
+  const maximum = counterpart('InputStatus.maximum')
+  const of = counterpart('InputStatus.of')
+  const required = counterpart('InputStatus.required')
   return (
     <StyledInputStatus>
       {count}{' '}
       {(!!tool.min || !!tool.max) &&
-        'of '}
+        `${of} `}
       {!!tool.min &&
-        <span style={{ color: minStyleColor }}>{tool.min} required</span>}
+        <span style={{ color: minStyleColor }}>{`${tool.min} ${required}`}</span>}
```

Comment:
Let's use Grommet's `Text` component here, which renders as a span, then we can style it using the color prop. Also, we should check with Becky on these colors. The red is hard to read to me when the input is selected and has the teal background.